src/OpenSage.Game/Logic/Orders/OrderProcessor.cs

```diff
@@ -61,30 +89,80 @@ public void Process(IEnumerable<Order> orders)
 
                     case OrderType.SetSelection:
                         // TODO: First argument is an unknown boolean.
+                        try
+                        {
+                            var objectIds = order.Arguments.Skip(1)
+                                .Select(x => (int) x.Value.ObjectId)
+                                .Select(_game.Scene3D.GameObjects.GetObjectById)
+                                .ToArray();
 
-                        var objectIds = order.Arguments.Skip(1)
-                            .Select(x => (int) x.Value.ObjectId)
-                            .Select(_game.Scene3D.GameObjects.GetObjectById)
-                            .ToArray();
+                            _game.Selection.SetSelectedObjects(player, objectIds);
+                        }catch(System.Exception e)
```

Comment:
i think we usually put the catch into a newline

Reply:
Fixed that